### ISSUE TITLE
Use icons of Discord and Matrix 

### ISSUE THREAD

**Issue by GrimPixel** [drwhut/tabletop-club#278]
The current icons are not intuitive and requires mouse hovering.

**Comment by drwhut** [comment]
While I would like to use them, I'm not 100% sure if I *can*, especially for the Discord logo. I was not able to find anything online as to how I can use the Discord logo, so I took the cautious route and used generic icons instead. I'm assuming Matrix is a little more relaxed in this regard, but at that point I decided to make the icons consistent.

With v0.2.0 the main menu will probably be re-worked anyway, my thinking being that the bottom of the main menu is starting to get cluttered. One idea I had was to put all of the external links in a pop-up window, with the titles being alongside the icons rather than in the tooltip.

**Comment by GrimPixel** [comment]
Ah, I see, you just don't have ideas about the law. Last time was using the system font.
If you visit a random site, you can see they are using icons of social media. That is nothing to fear.

**Comment by elmodor** [comment]
Can you use fontawesome?
They have a discord icon:
https://fontawesome.com/icons/discord

**Comment by GrimPixel** [comment]
https://commons.wikimedia.org/wiki/Category:Matrix.org_logos
https://commons.wikimedia.org/wiki/Category:Discord_(software)

**Comment by drwhut** [comment]
> Ah, I see, you just don't have ideas about the law. Last time was using the system font.
> If you visit a random site, you can see they are using icons of social media. That is nothing to fear.

It's not that I have no idea about the law (I had to take a course about software law at university), it's that I'm choosing to be strict about what I include with the base game. At the bare minimum, in order for an asset to be included in the base game, it needs to have an open license (CC0, CC BY-SA, etc.), which Wikimedia *claims* the discord logo is, but that might not necessarily be the case. **I would need to see Discord themselves saying the logo is open to use without restriction.**

This also doesn't account for the fact that the logo is trademarked, which is a whole other can of worms that I am not willing to open, which is why even if the icon is included in Font Awesome Free, I'm still anxious about using it.

In almost every single game I've played that has had a link to the game's Discord, it has been a text button using the game's font saying "Discord" or "Join us on Discord" or something along those lines - this will most likely be the route that I take in the future, since it does not open up a risk of infringing someone's trademark.

**Comment by GrimPixel** [comment]
There it is https://discord.com/branding
No statement for the Matrix logo.

**Comment by drwhut** [comment]
Of course, their branding guidelines explain to excrutiating detail how the logo should *look*, but not how it should be *used*... I would assume from their playful branding that they are pretty relaxed about it, but concrete wording would be nice ><

I think replacing the links at the bottom of the main menu with generic icons + text is the best way forward, which would solve the original issue of the icons by themselves not being descriptive enough.
